### PR TITLE
FNS App Frontend

### PR DESCRIPTION
**Pull request by SingingApple**

### CODE REVIEW

**Review by rekpero — CHANGES_REQUESTED** [review]
lgtm, just update the depedency

File: .yarnrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+yarn-path ".yarn/releases/yarn-1.19.0.cjs"
```
Comment (rekpero) [inline .yarnrc:5]: Can you remove this file?

**Review by rekpero — CHANGES_REQUESTED** [review]

File: packages/frontend/src/Routes.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { BrowserRouter, Routes, Route } from 'react-router-dom'
```
Comment (rekpero) [inline packages/frontend/src/Routes.tsx:2]: use hashrouting

**Review by rekpero — CHANGES_REQUESTED** [review]

**Review by rekpero — APPROVED** [review]

**Review by L-tech — APPROVED** [review]
LGTM

**Pull request merged**